packages/panna-sdk/src/core/utils/utils.ts

```diff
@@ -207,3 +213,163 @@ export const accountBalanceInFiat = async function (
 
   return result;
 };
+
+/**
+ * Get the total fiat value of multiple tokens and individual token balances
+ * @param params - Parameters for getting multiple account balances
+ * @returns Total value and individual token balances with fiat values, plus any errors
+ * @throws Error if address is invalid
+ * @example
+ * ```ts
+ * // Get portfolio value for multiple tokens
+ * const result = await accountBalancesInFiat({
+ *   client: pannaClient,
+ *   address: '0x...',
+ *   chain: lisk,
+ *   tokens: [
+ *     NATIVE_TOKEN_ADDRESS, // Native token
+ *     '0x...', // USDC
+ *     '0x...'  // DAI
+ *   ],
+ *   currency: 'USD'
+ * });
+ * // result: {
+ * //   totalValue: { amount: 5250.75, currency: 'USD' },
+ * //   tokenBalances: [
+ * //     {
+ * //       token: {
+ * //         address: undefined, // Native token
+ * //         symbol: 'ETH',
+ * //         name: 'Ethereum',
+ * //         decimals: 18
+ * //       },
+ * //       tokenBalance: {
+ * //         value: 2000000000000000000n, // 2 ETH in wei
```

Comment:
As a standard practice, it would nice to return the value as a string instead of a BigInt instance.